packages/react-devtools-shared/src/backend/views/Highlighter/index.js

```diff
@@ -74,7 +74,7 @@ export default function setupHighlighter(
       window.removeEventListener('mouseover', onMouseEvent, true);
       window.removeEventListener('mouseup', onMouseEvent, true);
       window.removeEventListener('pointerdown', onPointerDown, true);
-      window.removeEventListener('pointerover', onPointerOver, true);
+      window.removeEventListener('pointermove', onPointerMove, true);
```

Comment:
The last time I debugged it, `pointerover` was emitted each time when I hovered over some element inside shadow-root with mode = open.

I will double-check that tomorrow and return with more feedback

Reply:
Yes, in case when multiple components are inside shadow root, hovering over them will not fire multiple `pointerover` events.

Overall, looks good to me, thanks for working on this. I will leave 1 small comment, can merge after this changed

Reply:
thanks @hoxyq